apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -906,6 +925,42 @@ class ScriptEditor extends React.Component {
               }
             />
           </label>
+          <div>
```

Comment:
it might be better to hide the button entirely if this is not a PLC course

Reply:
Ya I thought about this but wondered if it was better to have the button there with the helptip to guide you into setting the right things if you want to launch